File: tests/fixtures/zig/scope.zig
const dependency = @import("dependency.zig");

fn helper(value: u32) void {
    dependency.consume(value);
}

fn run(loader: *const fn () u32) void {
    const value = loader();
    const callback = helper;
    callback(value);
    helper(value);
    missing(value);
}

File: tests/fixtures/elixir/scope.ex
defmodule Fixtures.Scope do
  alias App.Worker

  def helper(value), do: normalize(value)

  def run(loader, items) do
    callback = fn value -> helper(value) end
    local = loader.()

    Enum.map(items, fn {key, value} -> transform(key, value, local) end)
    callback.(Worker.build())
    missing()
  end
end
